docs/configure/babel.md

```diff
@@ -2,7 +2,9 @@
 title: 'Babel'
 ---
 
-Storybook’s webpack config by [default](#default-configuration) sets up [Babel](https://babeljs.io/) for ES6 transpiling. Storybook works with evergreen browsers and IE11 by default.
+Storybook’s webpack config by [default](#default-configuration) sets up [Babel](https://babeljs.io/) for ES6 transpiling. Storybook works with evergreen browsers by default.
+
+If you want to run Storybook in IE11, make sure to remove the `@storybook/addon-docs` addon, as this currently [causes issues in IE11](https://github.com/storybookjs/storybook/issues/8884). It's included in `@storybook/addon-essentials` which we add by default. So, make sure to replace this one with the individual addons you want to have.
```

Comment:
A couple points.

First, we don't recommend replacing addon-essentials with individual addons. you can disable individual addons in addon-essentials: https://storybook.js.org/docs/react/essentials/introduction#disabling-addons

Second, are you sure addon-docs breaks IE support? I thought this was fixed in 6.0 cc @tooppaaa

Reply:
It should have been but I think the root cause is highlight.js who drops support and isn't detected by are-you-es5

Reply:
If all of you are ok, tomorrow (wednesday) i can boot up my windows box and see if that's actually the case and report back with my findings. Sounds good?

Reply:
Sorry for not answering, but my windows box hd "was on the fritz" (pardon the bad pun) and i had to replace it, took a bit more time that expected. But i checked this during the weekend and i'm with @shilman on this. Instead of removing it, we could go with the instructions to disable the specific addon if ie11 is being used.